.github/workflows/test.yml

```diff
@@ -100,3 +100,41 @@ jobs:
     runs-on: ubuntu-22.04
     steps:
       - run: 'echo "No build required"'
+
+  e2e:
```

Comment:
Maybe worth naming things in anticipation for more of them?

Alternatively, move the e2e tests to a new workflow where we can isolate them (and name them) better; would also give a better at-an-glace feedback in the PR pipeline status that E2E was their own "row" from unit tests; since their domains and feedback loops are slightly different 

```suggestion
  e2e-github:
```